src/components/common/TmBalance.vue

```diff
@@ -82,6 +82,16 @@ export default {
     totalRewards() {
       return this.distribution.totalRewards[this.bondDenom]
     },
+    // only be ready to withdraw of the validator rewards are loaded and the user has rewards to withdraw
+    // the validator rewards are needed to filter the top 5 validators to withdraw from
+    readyToWithdraw() {
+      return (
+        Object.values(this.distribution.rewards)
+          .map(rewards => rewards[this.bondDenom])
+          .filter(bondDenomRewards => bondDenomRewards > 0).length > 0 &&
```

Comment:
I feel we should have a getter which computes a list of validators with non zero rewards. It could be used elsewhere